data-model/fdo-type/term/term.json

```diff
@@ -0,0 +1,148 @@
+      "type": "string",
+      "enum": [
+        "ods:Draft",
+        "ods:Active",
+        "ods:Tombstone"
+      ],
+      "description": "The status of the Digital Object",
+      "examples": [
+        "ods:Active"
+      ]
+    },
+    "ods:version": {
+      "type": "integer",
+      "description": "The version of the object, each change generates a new version. The version starts at 1 and each change will increment the version number with 1",
+      "minimum": 1,
+      "examples": [
+        1
+      ]
+    },
+ "ods:hasAgents": {
+      "type": "array",
+      "description": "Contains all agents that are connected to the specimen. Agents that are part of a specific part of the specimen should be added there. For example, a Collector is connected to the CollectingEvent so is added in the event.",
+      "items": {
+        "type": "object",
+        "$ref": "https://schemas.dissco.tech/schemas/fdo-type/shared-model/0.4.0/agent.json"
+      }
+    },
+    "dcterms:title": {
+      "type": "string",
+      "description": "A Name given to the resource, in this case openDS term name",
+      "examples": [
+        "ods:DigitalSpecimen"
+      ]
+    },
+    "skos:prefLabel": {
+      "type": "string",
+      "description": "The preferred lexical label for the term",
+      "examples": [
+        "Digital Specimen"
+      ],
+      "@language": {
+        "type": "string"
+      }
+    },
+    "skos:definition": {
+      "type": "string",
+      "description": "A definition of what the term represents.",
+      "examples": [
+        "A digital representation of a physical specimen"
+      ],
+  "@language": {
+        "type": "string"
+      }
+    },
+    "skos:note": {
+      "type": "string",
+      "description": "A note field to capture contexts and other information around the decision.",
+      "examples": [
+        "This term was discussed in projectX"
+      ],
+      "@language": {
+        "type": "string"
+      }
+    },
+  "skos:example": {
+      "type": "string",
+      "description": "An example usage of the term.",
+      "examples": [
+        "A record of a butterfly specimen collected in the field"
+      ]
+    },
+   
+    "dcterms:created": {
+      "type": "string",
+      "description": "The timestamp that the object was created in DiSSCo, following the ISO Date Time Format yyyy-MM-dd'T'HH:mm:ss.SSSXXX",
+      "format": "date-time",
+      "examples": [
+        "2023-10-02T12:31:34.806Z"
+      ]
+    },
+    "dcterms:modified": {
+      "type": "string",
+      "description": "The timestamp that the object was last changed, following the ISO Date Time Format yyyy-MM-dd'T'HH:mm:ss.SSSXXX",
+      "examples": [
+        "2023-10-02T12:31:34.806Z"
+      ]
+    },
+  
+    "dcterms:hasVersion": {
+      "type": "string",
+      "description": "A related resource that is a version, edition, or adaptation of the described resource.",
+      "examples": [
+        "1.0.1"
+      ]
+    },
+    "rdfs:range": {
```

Comment:
Shouldn't this be always present and therefore required? When is a term not part of a class?